### ISSUE TITLE
strange behavior with pxbp on computing cluster 

### ISSUE THREAD

**Issue by kateuckele** [FePhyFoFum/phyx#174]
Pxbp version 1.3. I believe phyx was binary compiled, but I was not the person who did this. All the phyx commands that I have used up until now have not produced any issues, that I know of. 

Running on computational cluster - CentOS Linux release 7.9.2009 (Core)

Clade species are missing for many *.tre.bp files, e.g., 

pxbp -t example.tre -o example.tre.bp
produces the following example output: 

621 trees 
4 unique clades found
CLADE:          FREQ:   0.539452        ICA:    0.08199 COUNT:  335     TREEFREQ:       0.539452
CLADE: Costus_dirzoi_98079 Costus_chartaceus_19269 Costus_zingiberoides_98162   FREQ:   1.      ICA:    1.      COUNT:  621     TREEFREQ:       1.
CLADE: Costus_zingiberoides_98162       FREQ:   0.243156        ICA:    -0.08199        COUNT:  151     TREEFREQ:       0.243156
CLADE: Costus_chartaceus_19269          FREQ:   0.217391        ICA:    -0.08199        COUNT:  135     TREEFREQ:       0.217391
TSCA: 1.24597

When pxbp is run on the exact same files on my local computer (MacOS V. 12.2.1) where phyx was installed with homebrew, it works as expected.

**Comment by josephwb** [comment]
Can you provide the files?

**Comment by kateuckele** [comment]
Sorry for the delay! Here is an example of a .tre file and its corresponding output .bp file
[test3.rooted_trees.tre.bp.txt](https://github.com/FePhyFoFum/phyx/files/9913671/test3.rooted_trees.tre.bp.txt)
[test3.rooted_trees.tre.txt](https://github.com/FePhyFoFum/phyx/files/9913672/test3.rooted_trees.tre.txt)

**Comment by josephwb** [comment]
Thanks.
I'm not sure exactly what the problem is. Is it that the clade members are not listed in the first clade? What are your expected results?

**Comment by josephwb** [comment]
If this solves things, then perhaps you have 2 different versions of phyx, with each one having a different default value for verbosity.

Please try re-running the example files using the verbose flag and let me know if things are working correctly.

**Comment by kateuckele** [comment]
I don't think that worked, but you are right that I'm using two versions of phyx. I'm using version 1.2 on my local computer, and I'm using 1.3 on the remote machine. I've attached a few output files for you. I've identified a potentially new issue that the v.1.3 output doesn't match the v.1.2 output (w/ same input tree files). While the verbose v.1.3 version matches the non-verbose v.1.3 version, both of the v.1.3 versions do not match the v.1.2 version. As one example, the v.1.3 versions produce 6 unique clades, and the v.1.2 version produces 4. Also, in the v.1.3 verbose version, the taxa included in the first clade is still missing (which is the initial issue that I wrote about). 

"test3.rooted_trees.tre.bp.v.1.3.txt" is the output from running `pxbp -t test3.rooted_trees.tre -o test3.rooted_trees.tre.bp` on the remote machine with pxbp 1.3
[test3.rooted_trees.tre.bp.v.1.3.txt](https://github.com/FePhyFoFum/phyx/files/9923554/test3.rooted_trees.tre.bp.v.1.3.txt)

"test3.rooted_trees.tre.bp.verbose.v.1.3.txt" is the output from running `pxbp -t test3.rooted_trees.tre -o test3.rooted_trees.tre.bp -v` on the remote machine with pxbp 1.3
[test3.rooted_trees.tre.bp.verbose.v.1.3.txt](https://github.com/FePhyFoFum/phyx/files/9923556/test3.rooted_trees.tre.bp.verbose.v.1.3.txt)

"test3.rooted_trees.tre.bp.v.1.2.txt" is the output from running `pxbp -t test3.rooted_trees.tre -o test3.rooted_trees.tre.bp` on my local machine with pxbp 1.2
[test3.rooted_trees.tre.bp.v.1.2.txt](https://github.com/FePhyFoFum/phyx/files/9923565/test3.rooted_trees.tre.bp.v.1.2.txt)

Here is the input tree file
[test3.rooted_trees.tre.txt](https://github.com/FePhyFoFum/phyx/files/9923579/test3.rooted_trees.tre.txt)
